src/issuers-verifier.ts

```diff
@@ -84,7 +84,7 @@ const verifyMethod: VerifierType["verify"] = async (document) => {
       VerifyAllowedIssuersCode.UNSUPPORTED_V3_DOCUMENT,
       VerifyAllowedIssuersCode[VerifyAllowedIssuersCode.UNSUPPORTED_V3_DOCUMENT]
     );
-  } catch (e) {
+  } catch (e: any) {
```

Comment:
why any ?

Reply:
use type guard and handle it as an Error :)

Reply:
Fixed 👍🏻